README.md

```diff
@@ -1,192 +1,165 @@
-
-#### add `mat/war/WEB-INF/jetty-env.xml`
-```xml
-<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure.dtd">
-<Configure id="Server" class="org.eclipse.jetty.webapp.WebAppContext">
-<New id="MatJndi" class="org.eclipse.jetty.plus.jndi.Resource">
-  <Arg></Arg>
-  <Arg>java:/comp/env/jdbc/mat_app_tomcat</Arg>
-  <Arg>
-    <New class="com.mysql.jdbc.jdbc2.optional.MysqlConnectionPoolDataSource">
-        <Set name="User">username</Set>
-        <Set name="Password">password </Set>
-        <Set name="DatabaseName">schemaname</Set>
-        <Set name="ServerName">serveraddress</Set>
-        <Set name="PortNumber">port</Set>
-      </New>
-    </Arg>
-</New>
-</Configure>
-```
-If you get exception as below `'com.google.gwt.dev.shell.jetty.JettyLauncher.WebAppContextWithReload'` is not of type `'org.eclipse.jetty.webapp.WebAppContext'` then you will need to find a way to load the jetty class instead of `gwt-dev.jar`. One way to make it work is to change the `gwt-dev` dependency in the `pom.xml` to be `provided` rather than `compile`. If you do this, then when running `mvn clean install` you will have to change it back.
-```xml
-<dependency>
-    <groupId>com.google.gwt</groupId>
-    <artifactId>gwt-dev</artifactId>
-    <version>${gwtVersion}</version>
-    <scope>provided</scope>
-    <exclusions>
-        <exclusion>
-            <artifactId>commons-collections</artifactId>
-            <groupId>commons-collections</groupId>
-        </exclusion>
-        <exclusion>
-            <groupId>org.eclipse.jetty</groupId>
-            <artifactId>apache-jsp</artifactId>
-        </exclusion>
-    </exclusions>
-</dependency>
+- Install MySQL (MAT currently has been tested with MySQL Community Version 8.0+) available from [MySQL] (https://dev.mysql.com/downloads/mysql/)
+- Run the MySQL community server installer for your operating system and the MySQL workbench (which comes with the download).
+- For Mac:
+- Go to System Preferences/ MY SQL after installing.
+- Click Initialize Database.
+- Enter a passowrd and click on Use Legacy Password Encryption. (Remember the username/pwd you will need these in future steps.)
+- Start the MYSQL Db.
+- In MeasureAuthoringTool/pom.xml I had to change the driver to match the community version I downloaded. I am always careful not to commit this change.
+  
 ```
+      <dependency>
+          <groupId>mysql</groupId>
+          <artifactId>mysql-connector-java</artifactId>
+          <version>8.0.19</version>
+      </dependency>
+```      
+- Create a new MySQL Connection to the database. I use jetbrains (datagrip)[https://www.jetbrains.com/datagrip/]
+- From the MAT Code base, find the `scripts/Dump*.sql` file and then execute the script in the database that was just created. <br> **(Note this script is from a dump and drops and create a schema called  `MAT_APP_BLANK`)**
```

Comment:
The current dump name is different. It's Dump_MAT_APP_BLANK_V6.sql
Also, there is another recommended dump on a shared google drive https://drive.google.com/drive/u/0/folders/1x0WhhIM9WIwCzXtmm46iF6wjXeVui5ct